src/rules/utils.ts

```diff
@@ -6,6 +6,7 @@ import {
   TSESTree,
 } from '@typescript-eslint/utils';
 import { version } from '../../package.json';
+import { isTypeOfJestFnCall } from './utils/parseJestFnCall';
```

Comment:
Rename _this_ file to `utils/index.ts`?

Reply:
I was planning on moving everything else in a follow-up PR, as I'll split things across a few more files but wanted to wait until I'd refactored `expect` parsing since that represents about 1/3rd of the file.